plugins/modules/selinux.py

```diff
@@ -28,6 +28,11 @@
     required: true
     choices: [ disabled, enforcing, permissive ]
     type: str
+  update_kernel_param:
+    description:
+      - If set to true, will update also the kernel boot parameters when disabling/enabling SELinux.
+      - The 'grubby' tool must be present on the target system for this to work.
+    type: bool
```

Comment:
```suggestion
    default: no
    type: bool
```

Hi @WOnder93!
I have checked the failed result of CI tests(https://dashboard.zuul.ansible.com/t/ansible/build/acc34e4373354f94959cf48d2b8be439).
```
ERROR: Found 1 validate-modules issue(s) which need to be resolved:
ERROR: plugins/modules/selinux.py:0:0: doc-default-does-not-match-spec: Argument 'update_kernel_param' in argument_spec defines default as (False) but documentation defines default as (None)
See documentation for help: https://docs.ansible.com/ansible/devel/dev_guide/testing/sanity/validate-modules.html
```

In this case, you probably need to specify the default value. Can you try to add `default: no` in the `update_kernel_param:` section?

Reply:
Of course, thanks! I forgot to check the CI results again... It sucks that I don't get email notifications when the CI finishes :/